packages/@webex/plugin-meetings/src/breakouts/request.ts

```diff
@@ -25,7 +25,7 @@ export default class BreakoutRequest extends StatelessWebexPlugin {
   }: {
     url: string;
     message: string;
-    options?: object;
```

Comment:
Ideally, it would be to avoid `object`, as it's less flexible than `Record<string, any> ` in short. Here is some discussion about that: https://stackoverflow.com/a/66575880